### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Feliverse**
### To-Do list: list structure
**Learning objectives**

- Use webpack to bundle JavaScript.
- Learn how to use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
-    element.innerHTML = _.join(['Hello', 'webpack - JS'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
+  const element = document.createElement('div');
+  return element;
+}
```
Comment (Felix45) [inline src/index.js]: - [ Required ] Please note that the project requirements state that the list of `Todos` should be generated dynamically using JavaScript, currently the `todo list` on your webpage uses plain HTML. Please see the screenshot below and rectify this.

## Project requirements
Screenshot from 2023-01-18 21-41-35

- You can approach this by following the steps below
  -  Create an array  of objects containing all tasks
  - Iterate over the tasks array and populate the HTML list item element for each task.

```JavaScript
const tasks = [
  {
     index: 0,
     completed: false,
     description: 'An example description in string',
  },
  {
     index: 1,
     completed: false,
     description: 'An example description in string',
  },
];

const printTodoList = () => {
  const listContainer = document.querySelector('.todo-list');
  tasks.forEach((todo) => {
    listContainer.innerHTML += `<li>
      <input type='checkbox' />
      <input type='text'  value='${todo.description}' />
      <span class='fa fa-ellipsis-v fa-lg'></span>
    </li>`;
  });
};
printTodoList();
```
Please note this is just an example, you don't have to implement it in the same way.

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Feliverse , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [ ] No linter errors :heavy_check_mark: 
- [ ] Descriptive pull request title and summary :heavy_check_mark: 
- [ ] GitHub flow is correctly used :heavy_check_mark: 
- [ ] Professional readme file :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Feliverse) [reply src/index.js]: Thank you @Felix45, 
my project looks good, I implement using the example you bring me.

**Review by BertrandConxy — APPROVED** [review]
Hi @Feliverse, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**